base-notebook/Dockerfile

```diff
@@ -26,109 +24,31 @@ RUN apt-get update --yes && \
     #   the ubuntu base image is rebuilt too seldom sometimes (less than once a month)
     apt-get upgrade --yes && \
     apt-get install --yes --no-install-recommends \
```

Comment:
A few lines above you upgrade all the packages and set DEBIAN_FRONTEND.
This is not needed anymore, because you've already done that in the `jupyter-base` image.